src/main/java/frc/trigon/robot/subsystems/collector/simulationcollector/SimulationCollectorConstants.java

```diff
@@ -0,0 +1,80 @@
+package frc.trigon.robot.subsystems.collector.simulationcollector;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import frc.trigon.robot.motorsimulation.MotorSimulationConfiguration;
+import frc.trigon.robot.motorsimulation.SimpleMotorSimulation;
+
+public class SimulationCollectorConstants {
+    static final boolean
+            FOC_ENABLED = true;
+    private static final int
+            ANGLE_MOTOR_AMOUNT = 1,
+            COLLECTION_MOTOR_AMOUNT = 1;
+    private static final double
+            ANGLE_MOTOR_GEAR_RATION = 1,
+            COLLECTION_MOTOR_GEAR_RATIO = 1;
+    private static final double
+            ANGLE_MOTOR_MOMENT_OF_INERTIA = 0.0001,
+            COLLECTION_MOTOR_MOMENT_OF_INERTIA = 0.0001;
+    private static final DCMotor
+            ANGLE_GEARBOX = DCMotor.getKrakenX60Foc(ANGLE_MOTOR_AMOUNT),
+            COLLECTION_GEARBOX = DCMotor.getKrakenX60Foc(COLLECTION_MOTOR_AMOUNT);
+    private static final double
+            ANGLE_VOLTAGE_COMPENSATION_SATURATION = 12,
+            COLLECTION_VOLTAGE_COMPENSATION_SATURATION = 12;
+    private static final double
+            ANGLE_MOTOR_CONVERSION_FACTOR = 360,
```

Comment:
Although it's easier to just say 360 here and get all the values in degrees, we still want to match the onboard pid calculation, which will be in 1 revolutions. You don't really need to have constants for things you're pretty much setting as default. You can have 1 conversions factor for all motors here